pkg/sql/sql.y

```diff
@@ -102,6 +102,9 @@
 
 	func attrsUnion(as1, as2 attrs) attrs {
 		for k, v := range as2 {
+			if _, ok := as1[k]; ok {
+				log.Panicf("attr %q already specified", as2)
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#dont-panic, do not panic, maybe just log the error and overwrite the attribute?

Reply:
I prefer using panic. We need to return the error of duplicated attributes to the user. Logging the error only shows up on the SQLFlow server.

Reply:
Well, `panic` will cause the server process exit, which we do not prefer because the server is a long-running process, we can not assume that user always input valid SQL statements. When the server exits, we need to manually restart it or auto restart it by Kubernetes, even it's restarted by Kubernetes, it will cause:

- temporary service down-grade
- broke currently handling incoming requests in other goroutines
- give more load to the Kubernetes master

and this kind of error better to return to the client, so maybe just put this in `ir_generator.go`?

Reply:
> Well, panic will cause the server process exit, which we do not prefer because the server is a long-running process, we can not assume that user always input valid SQL statements. 

There is a `recover` function for the parsing. So the server will not exit.

https://github.com/sql-machine-learning/sqlflow/blob/1cc07f0e3d611adec889c59773236873ee04898b/pkg/sql/parser.go#L316-L324

Reply:
I see, approving.